src/library.js

```diff
@@ -1452,6 +1452,7 @@ LibraryManager.library = {
     if (clk_id) {{{ makeSetValue('clk_id', 0, 2/*CLOCK_PROCESS_CPUTIME_ID*/, 'i32') }}};
     return 0;
   },
+  gettimeofday__sig: 'ii',
```

Comment:
I think the sig of this function is `iii` (return an int and takes 2 pointers).

Reply:
May be I am wrong, but I can see only one argument `ptr` in the function gettimeofday ([here](https://github.com/emscripten-core/emscripten/blob/main/src/library.js#L1456))

Reply:
It take a second argument which is the timezone (which this version currently completely ignores I guess): https://man7.org/linux/man-pages/man2/settimeofday.2.html.

Reply:
Updated